src/amo/components/Addon/styles.scss

```diff
@@ -237,3 +242,22 @@
   //   display: none;
   // }
 }
+
+.Addon-get-firefox {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  width: max-content;
+
+  .Addon-get-firefox-privacy-link,
+  .Addon-get-firefox-privacy-link:active,
+  .Addon-get-firefox-privacy-link:link,
+  .Addon-get-firefox-privacy-link:visited,
+  .Addon-get-firefox-privacy-link:hover {
+    @include font-medium();
```

Comment:
According to [this comment](https://github.com/mozilla/addons-frontend/issues/5245#issuecomment-395912680), the font size should `11px` and the font weight `700`. Currently we have `12px` and `500`. I find the current text too prominent, especially compared to this screenshot:

![](https://camo.githubusercontent.com/97fb350b196ab49e54714139de7577584d3eb7da/68747470733a2f2f692e696d6775722e636f6d2f3830567a4f6f772e706e67)

Last but not least, I'd expect some sort of `text-decoration` on hover/active.

Reply:
Ah, thanks, I was looking at the wrong one in the spec.